modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,61 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during a scheduled maintenance window period.
+Maintenance windows can be scheduled to suit your calendar.
+When you have created a schedule, you can select tasks that can only be performed during a scheduled maintenance window.
+
+Maintenance windows require both a calendar, and a schedule.
+The calendar defines the date and time of your maintenance window events, including recurring events, and must be in [path]``ical`` format.
+The schedule uses the events defined in the calendar to create the maintenance windows.
+You must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar, before you can create the schedule.
+
+When you have created the schedule, you can assign it to clients that are registered to the {productname} Server.
+Clients that have a maintenance window assigned cannot run restricted actions outside of the scheduled maintenance window.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Restricted actions include:
+
+* Package installation
+* Client upgrade
+* Service pack migration
+* Highstate application (for Salt clients)
+
+Unrestricted actions are minor actions that are considered safe and are unlikely to cause problems on the client.
+Unrestricted actions include:
+
+* Package profile update
+* Hardware refresh
+* Subscribing to software channels
+
+
+Before you begin, you must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar.
+You can create [path]``ical`` files in your preferred calendaring tool, such as KOrganizer.
+// I don't know if this is correct. ---LKB 2020-07-21
+
+
+
+.Procedure: Creating a New Maintenance Calendar
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Calendar], and click btn:[Create].
+. In the [guimenu]``Calendar Name`` section, type a name for your calendar.
+. Either provide a URL to your [path]``ical`` file, or upload the file directly.
+. Click btn:[Create Schedule] to save your calendar.
```

Comment:
Hm, I think the button says `Create Calendar` (could be that an incorrect version was deployed on the test server).

![ss-cal](https://user-images.githubusercontent.com/1412268/88047796-7641c500-cb52-11ea-977a-d6f38615da04.png)

Reply:
That came from a video, I haven't tested much yet. Will come back to this.

Reply:
FYI: the test server is showing 404 as of 2020-07-22 17:00 UTC. Will have to come back to testing later.

Reply:
Still 404 at 2020-07-23 0600 UTC

Reply:
Ouch! Working on it now.

Reply:
(anyway, the correct button is `Create Calendar`, so we should put this one to the docs. if you discover, it says `Create Schedule` in some future version of uyuni, then it's a bug and (i will eat my hat))